The test config finds the build directory by looking for the file
'BUILD-STAMP'. However, the build system creates 'build-stamp' instead
so it will never be found on a case-sensitive filesystem.

--- a/src/tests/config.lsp.in	2021-02-01 09:59:46.000000000 -0300
+++ b/src/tests/config.lsp.in	2021-11-20 18:12:39.318173405 -0300
@@ -41,7 +41,7 @@
   (loop with root = (si::get-library-pathname)
         with lib-name = (format nil "../lib/ecl-~A/" (lisp-implementation-version))
         for base in (list root (merge-pathnames lib-name root))
-        when (or (probe-file (merge-pathnames "./BUILD-STAMP" base))
+        when (or (probe-file (merge-pathnames "./build-stamp" base))
                  (probe-file (merge-pathnames "./COPYING" base)))
         do (return base)))
 
